yt_dlp/extractor/churchofjesuschrist.py

```diff
@@ -0,0 +1,33 @@
+from .common import InfoExtractor
+
+
+class ChurchOfJesusChristIE(InfoExtractor):
+    _VALID_URL = r'https?:\/\/(?:www\.)?churchofjesuschrist\.org\/broadcasts\/(?:(?:watch\/)|(?:article\/(?:(?:\w|-)*?)\/(?:\d*?)\/(?:\d*?)\/))(?P<id>(?:\w|-)+)(?:\?)*?'
```

Comment:
remove unnecessary escapes. eg: `\/` => `/`
can be simplified. eg: `(?:(?:\w|-)*?)\/` => `[\w-]+/`